docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -59,3 +59,38 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.2.
 {{% /admonition %}}
+
+### OAuth role mapping enforcement
+
+<!-- AuthNZ -->
```

Comment:
We can fall back to the team, but we always want a specific person to be the point of contact if it's needed during the ongoing editing process. 

```suggestion
<!-- Jo Guerreiro, AuthNZ -->
```